index.html

```diff
@@ -66,7 +64,14 @@
                         <td>Student Program</td>
                         <td><a href="https://developers.google.com/community/gdsc/leads">GDSC</a></td>
                     </tr>
-
+                        
+                     <tr>
+                        <td>Hacktoberfest 2022 Letsgoo!</td>
+                        <td>October</td>
+                        <td>October :(</td>
+                        <td>Hackathon</td>
+                        <td><a href="hacktoberfest.com">More about Hacktoberfest!</a></td>
+                     </tr>
```

Comment:
good work with the gradient @Suga7772 but hacktoberfest has already been added to the list please add something else.

I would suggues you can add Student Programs like Github Campus Expert about internship oppurtunities like Microsoft Engage also Google apprenticeship....

there are so many to add :) kindly make a new PR with the neccessary changes